tests/ACL/ACLScannerTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace OCA\groupfolders\tests\ACL;
+
+use OC\Files\Storage\Temporary;
+use OCA\GroupFolders\ACL\ACLManager;
+use OCA\GroupFolders\ACL\ACLStorageWrapper;
+use OCP\Constants;
+use Test\TestCase;
+
+/**
+ * @group DB
```

Comment:
Why group DB?

Reply:
the test users cache so it needs db access